openwisp_radius/tests/test_admin.py

```diff
@@ -363,9 +366,12 @@ def test_radius_batch_save_model(self):
         response = self.client.post(add_url, data, follow=True)
         error_message = 'Ensure this value is greater than or equal to 1'
         self.assertTrue(error_message in str(response.content))
+        self._delete_file('test-prefix12.pdf')
```

Comment:
I had to manually delete the file `test-prefix12.pdf` because `radbatch.pdf.path` contains `test-prefix12_<random-string>.pdf` yet this file exists as well as the one with the random string, is this intended behavior or a bug?